api: remove ssl (#379)
tarantool/go-tarantool: README.md

by the method `Do` of object `conn` which is the object that was returned
by `Connect()`.

> ### Example with openssl

Reviewer: ```suggestion
### Example with encrypting traffic
```
https://www.tarantool.io/en/doc/latest/enterprise/security/#encrypting-traffic

Author: Updated.

> To use the SSL dialer a

Reviewer: ```suggestion
To use the OpenSSL dialer a
```

Author: Updated.

[`go-tlsdialer`](https://github.com/tarantool/go-tlsdialer) package should be
imported:

```go
package tarantool

import (
	"context"
	"fmt"
	"time"

	"github.com/tarantool/go-tarantool/v2"
	"github.com/tarantool/go-tlsdialer"
)

func main() {
	sslDialer := tlsdialer.OpenSslDialer{
		Address:     "127.0.0.1:3013", 
		User:        "test", 
		Password:    "test", 
		SslKeyFile:  "testdata/localhost.key",
		SslCertFile: "testdata/localhost.crt",
		SslCaFile:   "testdata/ca.crt",
	}
	opts := tarantool.Opts{
		Timeout: time.Second,
	}

	ctx, cancel := context.WithTimeout(context.Background(), 500*time.Millisecond)
	defer cancel()
	conn, err := tarantool.Connect(ctx, sslDialer, opts)
	if err != nil {
		fmt.Printf("Connection refused: %s", err)
	}

	data, err := conn.Do(tarantool.NewInsertRequest(999).
		Tuple([]interface{}{99999, "BB"}), 
	).Get()
	if err != nil {
		fmt.Printf("Error: %s", err)
	} else {
		fmt.Printf("Data: %v", data)
	}
}
```

> Note that SSL is enabled only on Tarantool Enterprise Edition 2.10 or newer.

Reviewer: ```suggestion
Note that [traffic encryption](https://www.tarantool.io/en/doc/latest/enterprise/security/#encrypting-traffic)
is enabled only on Tarantool Enterprise Edition 2.10 or newer.
```

Author: Updated.

by the method `Do` of object `conn` which is the object that was returned
by `Connect()`.

### Example with encrypting traffic

> To use the OpenSSL dialer a

Author: Updated.

[`go-tlsdialer`](https://github.com/tarantool/go-tlsdialer) package should be
imported:

```go
package tarantool

import (
	"context"
	"fmt"
	"time"

	"github.com/tarantool/go-tarantool/v2"
	"github.com/tarantool/go-tlsdialer"
)

func main() {
	sslDialer := tlsdialer.OpenSslDialer{
		Address:     "127.0.0.1:3013", 
		User:        "test", 
		Password:    "test", 
		SslKeyFile:  "testdata/localhost.key",
		SslCertFile: "testdata/localhost.crt",
		SslCaFile:   "testdata/ca.crt",
	}
	opts := tarantool.Opts{
		Timeout: time.Second,
	}

	ctx, cancel := context.WithTimeout(context.Background(), 500*time.Millisecond)
	defer cancel()
	conn, err := tarantool.Connect(ctx, sslDialer, opts)
	if err != nil {
		fmt.Printf("Connection refused: %s", err)
	}

	data, err := conn.Do(tarantool.NewInsertRequest(999).
		Tuple([]interface{}{99999, "BB"}), 
	).Get()
	if err != nil {
		fmt.Printf("Error: %s", err)
	} else {
		fmt.Printf("Data: %v", data)
	}
}
```

Note that [traffic encryption](https://www.tarantool.io/en/doc/latest/enterprise/security/#encrypting-traffic)

> is enabled only on Tarantool Enterprise Edition 2.10 or newer.

Author: Updated.

Now you need to pass `Dialer` as the second argument instead of URI.
If you were using a non-SSL connection, you need to create `NetDialer`.

> For SSL-enabled connections, use `tlsdialer.OpenSslDialer` of the `go-tlsdialer`

Reviewer: ```suggestion
For SSL-enabled connections, use `OpenSslDialer` from the `go-tlsdialer`
```

Author: Updated.
